oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -398,7 +394,7 @@ public static ExternalAccountCredentials fromStream(
    */
   @SuppressWarnings("unchecked")
   static ExternalAccountCredentials fromJson(
-      Map<String, Object> json, HttpTransportFactory transportFactory) {
+      Map<String, Object> json, HttpTransportFactory transportFactory) throws IOException {
```

Comment:
Is this throw still needed?

Reply:
Look like not anymore. IDE does not gray it out though...